examples/diagnostics/custom_diagnostic.rs

```diff
@@ -1,17 +1,23 @@
 //! This example illustrates how to create a custom diagnostic.
 
 use bevy::{
-    diagnostic::{Diagnostic, DiagnosticId, Diagnostics, LogDiagnosticsPlugin},
+    diagnostic::{Diagnostic, DiagnosticId, LogDiagnosticsPlugin},
     prelude::*,
 };
+use bevy_internal::diagnostic::{Diagnostics, RegisterDiagnostic};
 
 fn main() {
     App::new()
         .add_plugins(DefaultPlugins)
         // The "print diagnostics" plugin is optional.
         // It just visualizes our diagnostics in the console.
         .add_plugin(LogDiagnosticsPlugin::default())
-        .add_systems(Startup, setup_diagnostic_system)
+        // Diagnostics must be initialized before measurements can be added.
+        // In general it's a good idea to set them up in a "startup system".
```

Comment:
Is this still the case? Looks like it's now better to use `register_diagnostic`. Probably should remove this.